File: Beginner/cpp/1052.cpp
// Written by: Md. Maruf Sarker
// Platform: Beecrowd
// Problem: https://www.beecrowd.com.br/judge/en/problems/view/1052
// Language: cpp / go / c / python
// Date: 2022-09-26
// Difficulty Rating: 

#include <bits/stdc++.h>
using namespace std;

#define endl "\n"
#define pb push_back
typedef long long ll;
typedef unsigned long long ull;
typedef long double ld;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);  cout.tie(NULL);

	int n;
	scanf("%d", &n);

	switch (n) {
		case 1:
			printf("January\n");
			break;
		case 2:
			printf("February\n");
			break;
		case 3:
			printf("March\n");
			break;
		case 4:
			printf("April\n");
			break;
		case 5:
			printf("May\n");
			break;
		case 6:
			printf("June\n");
			break;
		case 7:
			printf("July\n");
			break;
		case 8:
			printf("August\n");
			break;
		case 9:
			printf("September\n");
			break;
		case 10:
			printf("October\n");
			break;
		case 11:
			printf("November\n");
			break;
		case 12:
			printf("December\n");
			break;
	}
	return 0;
}

File: Beginner/go/1052.go
// https://www.beecrowd.com.br/judge/en/problems/view/1052
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	switch n {
		case 1:
			fmt.Println("January")
		case 2:
			fmt.Println("February")
		case 3:
			fmt.Println("March")
		case 4:
			fmt.Println("April")
		case 5:
			fmt.Println("May")
		case 6:
			fmt.Println("June")
		case 7:
			fmt.Println("July")
		case 8:
			fmt.Println("August")
		case 9:
			fmt.Println("September")
		case 10:
			fmt.Println("October")
		case 11:
			fmt.Println("November")
		case 12:
			fmt.Println("December")
	}
}